docs/source/testing/testing.mdx

```diff
@@ -122,10 +117,9 @@ If you want to test the behavior of your `context` function directly, we recomme
 
 Instead of bypassing the HTTP layer, you might want to fully run your server and test it with a real HTTP client. Apollo Server doesn't provide built-in support for this at this time.
 
-Instead, you can run operations against your server using a combination of any HTTP or GraphQL client such as [`supertest`](https://www.npmjs.com/package/supertest) or [Apollo Client's HTTP Link](/react/api/link/apollo-link-http/) .
+Instead, you can run operations against your server using a combination of any HTTP or GraphQL client such as [`supertest`](https://www.npmjs.com/package/supertest) or [Apollo Client's HTTP Link](/react/api/link/apollo-link-http/).
 
-<!-- TODO(AS4) double check if this package is going to be updated for AS4 -->
-<!-- There are also community packages available such as [`apollo-server-integration-testing`](https://www.npmjs.com/package/apollo-server-integration-testing), which uses mocked Express request and response objects. -->
+There are also community packages available such as [`@apollo/server-integration-testsuite`](https://www.npmjs.com/package/@apollo/server-integration-testsuite), which provides a suite of Jest tests that is useful for [integration authors](../integrations/building-integrations).
```

Comment:
These packages aren't really counterparts. Calling out the new one _on_ the building integrations page makes sense, but this article pertains more to testing out your server as an end user.

We don't have an equivalent testing utility for AS4, and I don't expect this one to be compatible (though that's an assumption). I think we should just remove this callout for now.

Reply:
Oh no, I didn't see this till right now—> I addressed this in this PR: https://github.com/apollographql/apollo-server/pull/6915